ReactViewResources/package.json

```diff
@@ -9,7 +9,11 @@
     "react": "16.8.1",
     "react-dom": "16.8.1",
     "require-css": "0.1.10",
-    "requirejs": "2.3.6"
+    "requirejs": "2.3.6",
```

Comment:
require still here?

Reply:
It won't be necessary once we append the bundles to the html (dynamic loading part).

Reply:
Removed (still have to update Loader)